KotlinFutures/app/src/main/java/com/example/kotlinfutures/model/poko/Coordinates.kt

```diff
@@ -0,0 +1,12 @@
+package com.example.kotlinfutures.model.poko
+
+import com.google.gson.annotations.SerializedName
+
+data class Coordinates(
```

Comment:
Please use next implementation of response\request 
`data class AuthEntity(@SerializedName("latitude") private val latitude: String, @SerializedName("longitude") private val longitude: UserEntity)`